src/quest_manager/views.py

```diff
@@ -160,6 +160,10 @@ def form_valid(self, form):
         # Save the object so we can add prereqs to it
         super_response = super().form_valid(form)
         self.set_new_prereqs(form)
+
+        if form.cleaned_data.get("archived") is True:
```

Comment:
Perhaps a post-save signal is a cleaner way to do this?  Having this in a view feels clunky to me. Also, this will miss instances where archived is set to true via django admin.

We need to ensure this works for bulk archiving too, there is a Django admin action for this.

Reply:
done